client/src/components/layout/ShowService.js

```diff
@@ -0,0 +1,57 @@
+import React, { useState, useEffect } from 'react'
+import ReviewTile from './ReviewTile'
+
+const ShowService = (props) => {
+  const [service, setService] = useState({
+    name: '',
+    number: '',
+    email: '',
+    websiteUrl: '',
+    photoUrl: '',
+    rating: '',
+    reviews: []
+  })
+
+
+  const getServiceAndReviews = async () => {
+    try{
+      const response = await fetch(`/api/v1/services/${props.match.params.id}`)
+      if(!response.ok){
+        throw new Error(`${response.status} ${response.statusText}`)
+      }
+      const body = await response.json()
+      setService(body.service)
+    }catch (error) {
+      console.error(`Error in fetch: ${error}`)
+    }
+  }
+
+  useEffect(() => {
+    getServiceAndReviews()
+  }, [])
+
+  const reviews = service.reviews.map((review) => {
+    return <ReviewTile
+            key={review.id}
+            review={review}
+            />
+  })
```

Comment:
unfortunately I have to be a stickler. This indentation looks little wild. The props should be indented in by one, and you should only use `return` without parens if you can fit it all on one line. So choose one or the other